device-connectors/src/testflinger_device_connectors/devices/oem_autoinstall/oem_autoinstall.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, config, job_data):
             self.config = yaml.safe_load(configfile)
         with open(job_data, encoding="utf-8") as job_json:
             self.job_data = json.load(job_json)
+        self.data_path = (
+            Path(__file__).parent / "../../data/muxpi/oem_autoinstall"
```

Comment:
I just realized that this has been using files under data/muxpi, shouldn't it organize them under data/oem_autoinstall instead? That seems like a better place for it, since they are not used by the muxpi device connector.